reference/sodium/functions/sodium-bin2base64.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- $Revision$ -->
+<!-- EN-Revision: 14e3e35ca0c82b3b5fb3fd71b9dd70e9eb250ab1 Maintainer: joshuaruesweg Status: ready -->
+<refentry xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="function.sodium-bin2base64">
+ <refnamediv>
+  <refname>sodium_bin2base64</refname>
+  <refpurpose>Kodiert eine binäre Zeichenkette mit Base64.</refpurpose>
```

Comment:
No period in refpurpose and title.
```suggestion
  <refpurpose>Kodiert eine binäre Zeichenkette mit Base64</refpurpose>
```

Reply:
In English, there is also a point here. Then we should remove that there too, if necessary, so that it is uniform in all languages?

Reply:
At least this would comply with the [Style guidelines](http://doc.php.net/tutorial/style.php). But I don't want to open this can of worms. ;-)